src/theme/components/template.ts

```diff
@@ -4,6 +4,9 @@ import { defineStyle } from "@chakra-ui/system";
  * Grid layout based on https://www.joshwcomeau.com/css/full-bleed/
  */
 
+export const responsiveGutters = { base: "1rem", md: "1.5rem", xl: "1rem" };
```

Comment:
@bigfishdesign13, just checking – the gutters are meant to get bigger at the `md` breakpoint and smaller at `xl`?

Reply:
Yes, that was the intention. The `md: "1.5rem"` here aligns with the `md: "1.5rem"` in `responsiveSpacing`. But I'll admit, I was a bit wishy washy on this. Maybe it should just be `"1rem"` across the board.